src/app/icd/client/CheckInHandler.cpp

```diff
@@ -127,6 +128,7 @@ CHIP_ERROR CheckInHandler::OnMessageReceived(Messaging::ExchangeContext * ec, co
     else
     {
         mpCheckInDelegate->OnCheckInComplete(clientInfo);
+        mpImEngine->OnActiveModeNotification(clientInfo.peer_node);
```

Comment:
Where did the CHIP_CONFIG_ENABLE_READ_CLIENT ifdef go?  Won't this break builds?

Reply:
just add it back, thanks

Reply:
But now this is not addressing the previous review comment about this.

Reply:
just add right config #include <app/AppConfig.h>, thanks